### ISSUE TITLE
bootstrap3ElementModifier : Useless 'has-feedback' class when !addValidationStateIcons

### ISSUE THREAD

**Issue by elkami12** [jonsamwell/angular-auto-validate#100]
Hi,

First thanks for your work and sorry for my english...
I think that in bootstrap 3, 'has-feedback' class should not be added to form-group if we don't use validation state icons...

I think simply add a `|| !addValidationStateIcons` condition should do the job, no ?
```javascript
frmGroupEl.addClass('has-success ' + (inputGroupEl.length > 0 || !addValidationStateIcons ? '' : 'has-feedback'));
```
in `#makeValid` and the equivalent in `#makeInvalid`...

What do you think of that and could you add this fix ?

Thanks !

**Comment by jonsamwell** [comment]
Fixed in release v1.19.0 :-)

**Comment by dustin-page** [comment]
Thanks for the fix. I just ran into this issue and 'has-feedback' class was cutting off my text in the input when displaying errors.
